libenkf/src/enkf_main.c

```diff
@@ -469,13 +469,13 @@ member_config_type * enkf_main_iget_member_config(const enkf_main_type * enkf_ma
 
 
 
-void enkf_main_node_mean( const enkf_node_type ** ensemble , int ens_size , enkf_node_type * mean ) {
+static void enkf_main_node_mean( const vector_type * ensemble , enkf_node_type * mean ) {
   int iens;
```

Comment:
perhaps assert that `vector_get_size(ensemble) > 0` and even extract that variable out, and put `iens` into `for` definition?
```c
int ens_size = vector_get_size(ensemble);
if (ens_size == 0)
  util_abort(...);
for (int iens = 0; iens < ens_size; iens++)
  enkf_node_iadd(mean, vector_iget_const(ensemble, iens));
enkf_node_scale(mean, 1.0/ens_size);
```